npk/Cargo.toml

```diff
@@ -5,6 +5,7 @@ authors = ["ESRLabs"]
 edition = "2018"
 
 [dependencies]
+anyhow = "1.0.34"
```

Comment:
Why is anyhow needed in the `npk` crate? It has it's own error type and there's no need for anyhow

If it's just used in the tests it should be reference in `dev.dependencies`.